### PR TITLE
Reorganize chemical substance

### PR DESCRIPTION
**Pull request by vdancik**

### CODE REVIEW

File: biolink-model.yaml [hunk 1]
```diff
@@ -883,6 +886,14 @@ slots:
     exact_mappings:
       - WIKIDATA_PROPERTY:P274
 
+  is_metabolite:
```
Comment (deepakunni3) [inline biolink-model.yaml:889]: @vdancik This should be `is metabolite` (without the underscore).

Comment (deepakunni3) [reply biolink-model.yaml:889]: Also, is this slot supposed to hold a boolean value? i.e. `is_metabolite: true`?

If so, then the range should be `boolean` and the domain should be `chemical substance`

**Review by deepakunni3 — CHANGES_REQUESTED** [review]
Few notes on the PR

**Review by RichardBruskiewich — CHANGES_REQUESTED** [review]
Seems like you are transforming  `has active ingredient`  from an integral dedicated slot "property" of a drug, into a predicate of an association with the drug.  It may be helpful to review the applicable use cases to better understand whether this design is desirable (or not).

Note that if we keep this design  decision, then  perhaps  'has  excipient` (refers to the inactive components of a drug) should suffer the same fate?

**Comment by vdancik** [comment]
I agree that it makes sense to transform "has excipient" from a slot to a predicate and also add "is excipient of" inverse predicate.

**Comment by cmungall** [comment]
> I agree that it makes sense to transform "has excipient" from a slot to a predicate and also add "is excipient of" inverse predicate.

Agreed

**Review by RichardBruskiewich — APPROVED** [review]

**Pull request merged**